### PR TITLE
EPMCTSTAI-223

### PR DESCRIPTION
**Pull request by alex-popov-tech**

### CODE REVIEW

File: panels/CreatePromptPanelScripts.js [hunk 1]
```diff
@@ -1,125 +1,126 @@
...
+        deleteButton.textContent = "Delete"
+        deleteButton.setAttribute("id", idCounter)
+        deleteButton.onclick = () => {onDeleteVar(idCounter-3)}
+        idCounter += 1
+        console.log("gridCells[i]", gridCells[i], i)
+        gridCells[i].shadowRoot.appendChild(deleteButton)
+      }
+    }
+  }, 2000)
+}*/
```
Comment (KirillChernyshev) [inline panels/CreatePromptPanelScripts.js:41]: Maybe delete this as we are prettifying the code in the PR?

File: panels/CreatePromptPanelScripts.js [hunk 2]
```diff
@@ -1,125 +1,126 @@
...
+        >Delete</vscode-button>
+      </vscode-data-grid-cell>`;
+    dataGrid.append(row);
+    // dataGrid.rowsData = [...dataGrid.rowsData,
+    //   { id: 1,
+    //     name: varName,
+    //     value: varVal
+    //   }
+    // ];
+    // addDeleteButton()
```
Comment (KirillChernyshev) [inline panels/CreatePromptPanelScripts.js:76]: And this as well

File: panels/main.ts [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
+      name: "",
+      value: ""
+    }
+  ];
+  dataGrid.columnDefinitions = [
+    { columnDataKey: "name", title: "Name" },
+    { columnDataKey: "value", title: "Value" },
+    { columnDataKey: "delete", title: "" }
+  ];
+  */
```
Comment (KirillChernyshev) [inline panels/main.ts:19]: And this

**Review by KirillChernyshev — APPROVED** [review]
LGTM, thanks for formatting the code 🥇 

Would be awesome to add something to prevent doing it again. Precommit hook and/or event better add GitHub Action to check lint errors

**Review by MikalaiB — APPROVED** [review]

**Pull request merged**